apps/gdevelop/gdevelop.yml

```diff
@@ -0,0 +1,25 @@
+name: 'GDevelop'
+description: 'Create games without programming.'
+website: 'https://gdevelop-app.com/'
+repository: 'https://github.com/4ian/GDevelop'
+keywords:
+  - 'Game Editor'
+  - 'Open Source'
+  - 'Cross-Platform '
+  - 'Visual Programming'
+  - 'Online Editor'
+  - 'Piskel Sprite Editor'
+  - 'Google AdMob'
+license: MIT
+category: 'Graphics & Design'
+npmPackageName: 'gdmod'
```

Comment:
As owner of the [GDMod](https://github.com/arthuro555/gdmod) package I want to precise that even tho I am part of the moderation team of GDevelop and a GDevelop contributor this project is not affiliated officially with GDevelop.

Reply:
It seems `npmPackageName` is not required. Can we simply remove it?

Reply:
Surely that can be done :)